diagrams/ecosystem.svg

```diff
@@ -58,7 +58,7 @@
                             <font style="font-size:16px">
                                 <i>
                                     <b>
-                                        Holder
+                                        Issuee/Holder
```

Comment:
Based on the totality of conversation below, this label should revert.
```suggestion
                                        Holder
```
Additionally, the line below which now reads `Acquired, stores, presents VCs` should be changed to `Acquires, stores, presents VCs` ( `Acquired` becomes `Acquires`).

I have also just noticed, and am concerned that, this diagram no longer includes any transfers of VCs from `Holder` to `Holder` — only from `Holder` to `Verifier`. I think the `Holder` to `Holder` transfers need to be included here.